### PR TITLE
Add Subjects data in course_runs api

### PR DESCRIPTION
**Pull request by burhandodhy**
**Description**
This PR adds the new field `marketing_url` in `Subjects` model and also add the `subjects` data in `course_runs` API. 

**Jira Ticket**
https://edlyio.atlassian.net/browse/EDLY-2870

image

### CODE REVIEW

**Review by rehan99000 — APPROVED** [review]

**Review by iam-mhaseeb — APPROVED** [review]
👍 Don't we need to update tests for serializer?

**Comment by burhandodhy** [comment]
> 👍 Don't we need to update tests for serializer?

@iam-mhaseeb will update the tests later once all the sub-tasks gets completed.

**Review by etechdevai — APPROVED** [review]

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -863,6 +863,9 @@ class CourseRunSerializer(MinimalCourseRunSerializer):
         queryset=ProgramType.objects.all()
     )
     estimated_hours = serializers.SerializerMethodField()
+    subjects = SlugRelatedFieldWithReadSerializer(slug_field='slug', required=False, many=True,
```
Comment (zubair-arbi) [inline course_discovery/apps/api/serializers.py]:
```python
subjects = SlugRelatedFieldWithReadSerializer(
    slug_field='slug', 
    required=False, 
    many=True,
    queryset=Subject.objects.all(),
    read_serializer=SubjectSerializer(),
)
```

**Review by zubair-arbi — APPROVED** [review]
One small suggestion otherwise changes looks good 👍

File: course_discovery/apps/api/serializers.py (hunk 1 above)
Comment (burhandodhy) [reply course_discovery/apps/api/serializers.py]: Done

**Pull request merged**